solutions/usaco-832.mdx

```diff
@@ -0,0 +1,92 @@
+---
+id: usaco-832
+source: USACO Bronze 2018 Open
+title: Milking Order
+author: Ananth Kashyap
+---
+
+## Solution 1
+
+[Official Analysis](http://www.usaco.org/current/data/sol_milkorder_bronze_open18.html)
+
+## Solution 2
+
+In this problem, there are three cases that we have to consider: 
+- The first of these cases is when cow one is fixed. In this case, the solution is simple, as we just output its fixed position. 
+- In the second case, we must consider is if cow one is in the given hierarchy of cows. In this case, we want to place every cow in the hierarchy as early as possible. 
+- The last case is when cow one is neither fixed nor in the hierarchy. In this case, we would want to place every cow in the hierarchy as late as possible, allowing for cow one to get the earliest position possible.
+
+**Time Complexity:** $\mathcal{O}(N(M + N))$
+
+<LanguageSection>
+<PySection>
+
+## Python Implementation
```

Comment:
move this outside languagesection (and above the time complexity) and just put ## Implementation

@HarryWangATX i think you suggested this? the template on the guide just has ## Implementation, so i think we should stick with that
i think the reason is that the header should be above time complexity

Reply:
What should I do

Reply:
> move this outside languagesection (and above the time complexity) and just put ## Implementation

Reply:
```
## Implementation

**Time Complexity:** $\mathcal{O}(N(M + N))$

<LanguageSection>

<PySection>

... your code, etc ...
```